### DISCUSSION TITLE
Ant BMS

### DISCUSSION THREAD

**Discussion by NicoThien** [Louisvdw/dbus-serialbattery#127] [General]
Hello,
i have a problem. My battery consists of 11 LTO cells.
I adjusted the cell voltages in the "utils.py" file. The ANT-BMS was detected. The data displayed is incorrect.

**Comment by NicoThien** [comment]
I'm going crazy! After several restarts, the displayed data is correct. However, I now have an error message because the cell voltage is too low. However, this is within the parameters specified in utils.py.

Edit: My mistake. I guess I reinstalled the driver in the meantime. The original utils.py file was reinstalled with the wrong cell values.

**Reply by Louisvdw** [reply]
So with the utils.py adjusted again to your adjusted settings, is it then all working for you?

**Reply by NicoThien** [reply]
Many thanks for your great work.
Unfortunately, after restarting the raspberry, gibberish is displayed. Only after several reboots is it correct again. I haven't figured out how to display all cell voltages yet.

**Reply by NicoThien** [reply]
I had used the "SSH installer script option". Version 0.1 was installed. Now I have switched to V 0.11beta. Any reboot works now. Unfortunately no display of all cell voltages.
